web/template/watch.gohtml

```diff
@@ -12,6 +12,30 @@
     <link rel="stylesheet" href="/static/node_modules/video.js/dist/video-js.min.css">
     <link rel="stylesheet" href="/static/node_modules/videojs-seek-buttons/dist/videojs-seek-buttons.css">
     <link rel="stylesheet" href="/static/node_modules/@silvermine/videojs-airplay/dist/silvermine-videojs-airplay.css">
+
+    <link rel="stylesheet" href="//code.jquery.com/ui/1.13.1/themes/base/jquery-ui.css">
```

Comment:
Wir JQueryUI wirklich gebraucht? Wir verwenden fast überall Tailwind CSS. Wenn es möglich ist würde ich versuchen statt JqueryUI Tailwind und im notfall inline-css zu verwenden :).

Reply:
I am using the "interactions" Draggable and Resizable that is present in JQueryUI. I am pretty sure this is something completely different than Tailwind CSS. Another option would be to implement this from scratch, but it would take me quite some time to do that since I am not that knowledgeable in javascript. Do you think this can be accomplished with Tailwind CSS and inline-css only?

Reply:
and this stylesheet needs to be included for JQueryUI to work

Reply:
Yes, you would need some js too. Probably I am too biased as in my opinion it's never a good idea to use Jquery at all, but I think you can leave it like that if the others are okay with it 👍

Reply:
Also just as an idea there are some more modern libs for draggable components like https://interactjs.io/ which would be lighter and has support for touch devices :)

Reply:
I didn't know about these! I am reading up now, thanks for the suggestions :smiley: